### PR TITLE
Fixed the resources and Adjusted the sections , renamed

### PR DESCRIPTION
**Pull request by divpaul2016**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,159 +193,16 @@ <h6>St. John</h6>
...
     <!-- Section: About Us -->
     <section id="aboutus" class="home-section bg-gray paddingbot-60">
         <div class="container marginbot-50">
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2">
-                    <div class="wow fadeInDown" data-wow-delay="0.1s">
+                    <div class="wow lightSpeedIn" data-wow-delay="0.1s">
                         <div class="section-heading text-center">
-                            <h2 class="h-bold">About Us</h2>
+                            <h2 class="h-bold">About St.George</h2>
```
Comment (josephjoys) [inline index.html]: This is about Us right? Do we have anything about St.George in this page? May be About St. George Church?

File: index.html [hunk 2]
```diff
@@ -824,6 +642,140 @@ <h6 class="h-bold">Other Activities</h6>
...
+                  </div>
+                  <div class="col-sm-4 col-md-4">
+                      <div class="wow fadeInRight" data-wow-delay="0.1s">
+                          <div class="service-box">
+                              <div class="service-desc">
+                                  <h4 class="h-bold">Administration</h4>
+                                  <div>
+                                      <ul>
+                                          <li>
+                                              <a href="resources\StGeorgeChurch_AddressBook.xlsx" target="_blank"><i class="fa fa-download icon-primary"></i> Church Address Book <i>(Protected by password)</i></a>
```
Comment (josephjoys) [inline index.html]: the slash must be / not \

File: index.html [hunk 3]
```diff
@@ -824,6 +642,140 @@ <h6 class="h-bold">Other Activities</h6>
...
+                                  <div>
+                                      <ul>
+                                          <li>                                             
+                                               <a href="http://www.malankara.com/church/commonprayerbook.html" target="_blank"><i class="fa fa-download icon-primary"></i> Common Prayers</a>
+                                          </li>
+                                           <li>
+                      <a href="http://www.stgeorgechurchseattle.org/docs/Christmas%20Service%20Songs%20-%20Manglish.pdf" target="_blank"><i class="fa fa-download icon-primary"></i> Christmas Service Songs in Manglish</a>
+                    </li>
+                    <li>
+                      <a href="resources\The_Matrimony_Service_Book.docx" target="_blank"><i class="fa fa-download icon-primary"></i> The Matrimony Service Book</a>
```
Comment (josephjoys) [inline index.html]: change slash to /

File: index.html [hunk 4]
```diff
@@ -824,6 +642,140 @@ <h6 class="h-bold">Other Activities</h6>
...
+                              <div class="service-desc">
+                                  <h4 class="h-bold">Administration</h4>
+                                  <div>
+                                      <ul>
+                                          <li>
+                                              <a href="resources\StGeorgeChurch_AddressBook.xlsx" target="_blank"><i class="fa fa-download icon-primary"></i> Church Address Book <i>(Protected by password)</i></a>
+                                              
+                                          </li>
+                                          <li>
+                                              <a href="http://www.stgeorgechurchseattle.org/docs/St.George-Church-By%20Laws.pdf" target="_blank"><i class="fa fa-download icon-primary"></i> Church Bylaws</a>
```
Comment (josephjoys) [inline index.html]: this pdf must be downloaded and included in the resources folder

File: index.html [hunk 5]
```diff
@@ -824,6 +642,140 @@ <h6 class="h-bold">Other Activities</h6>
...
+                          </div>
+                      </div>
+                      <div class="wow fadeInRight" data-wow-delay="0.2s">
+                          <div class="service-box">
+                              <div class="service-desc">
+                                  <h4 class="h-bold">Prayers</h4>
+                                  <div>
+                                      <ul>
+                                          <li> 
+                                              <a href="http://www.stgeorgechurchseattle.org/docs/StGeorge_HolyWeek_Prayers.pdf" target="_blank"><i class="fa fa-download icon-primary"></i> Holy Week Prayers</a>
```
Comment (josephjoys) [inline index.html]: Download the file in to resources folder

File: index.html (hunk 1 above)
Comment (divpaul2016) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (divpaul2016) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (divpaul2016) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (divpaul2016) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (divpaul2016) [reply index.html]: Done

**Pull request merged**